components/match_ticker/commons/match_ticker_display_components_new.lua

```diff
@@ -0,0 +1,382 @@
+---
```

Comment:
mind depending on how different this is from the old display components (am on mobile until incl. monday, so shit to check) you could just extend the existing classes from the old module and hence save quite a bit of duplicate code (assuming there is much duplicate code)

Reply:
True, I didn't think about it as when talking to Rikard he told me to just make a new component for it. The final HTML structure is quite different however at the end so you can look at the new structure and make a call regarding what would be best

Reply:
lmk when it is ready and i can compare^^
(will mostly not be on PC tomorrow and Tuesday next week (and on weekends) apart from that i should be able to give a review within a day when requested^^)

Reply:
Now it is ready. Also, right now this calls the Countdown without many parameters since we want to be able toggle a switch to either show the date or the countdown of a match. That means that as of right now both elements will be rendered and I have a separate logic that hides the one that is not selected from the switch (will be in another MR).

This is not the ideal scenario I guess but it could be part of the Countdown standardization MR